pkg/identity/certificated.go

```diff
@@ -248,6 +248,10 @@ func Certificated(idConfig *config.IdentityConfig, stopChan <-chan struct{}) (er
 			}
 		}
 
+		if identity == nil || len(keyPEM) == 0 {
```

Comment:
I do not think this condition check is appropriate since we still expect that the `identity` and the `keyPem` may be empty when SIA intends to read identity certificate that was provided externally.
https://github.com/AthenZ/k8s-athenz-sia/blob/bca19beac47e2edf810237c07f91f726d99226db/pkg/identity/certificated.go#L162
This condition disables the capability for SIA to read the certificate which was provided externally.